docs/calendar-review.md

```diff
@@ -0,0 +1,214 @@
+# Supporting Calendar Systems in Temporal
+
+Authors: Temporal champions [justingrant](https://twitter.com/justingrantjg) and [sffc](https://twitter.com/_sffc)
```

Comment:
One more note: do we need to call out authors here?  We don't do that anywhere else in Temporal docs, so not sure there's a need for it here.

Reply:
Removed